libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.m

```diff
@@ -354,9 +354,9 @@ - (void)start {
         return;
     }
     
-    [self willChangeValueForKey:@"executing"];
```

Comment:
See KVO-compliant properties paragraph in https://developer.apple.com/library/mac/documentation/Cocoa/Reference/NSOperation_class/